cmd/agent/install_mac_os.sh

```diff
@@ -132,6 +132,13 @@ if [ -n "$DD_SYSTEMDAEMON_INSTALL" ]; then
     fi
 fi
 
+if [ "$systemdaemon_install" != false ]; then
+  if [ ! -e "/tmp/install-ddagent" ]; then
+    mkdir /tmp/install-ddagent
```

Comment:
I am not 100% sure `/tmp` is always present on MacOS. Did you check? Usually `$TMPDIR` on MacOS return something different than `/tmp`